vsdenoise/prefilters.py

```diff
@@ -90,9 +91,7 @@ def __call__(  # type: ignore
             pref_type = Prefilter.DFTTEST
 
         if pref_type == Prefilter.DFTTEST:
-            dftt_args = dict[str, Any](tbsize=1, slocation=[0.0, 4.0, 0.2, 9.0, 1.0, 15.0]) | kwargs
-
-            dfft = clip.dfttest.DFTTest(**dftt_args)
+            dfft = DFTTest.denoise(clip, tr=0, sloc=[(0.0, 4.0), (0.2, 9.0), (1.0, 15.0)], **kwargs)
```

Comment:
Maybe a bit outside the scope of the PR, but it might be a good idea to make sure `tr` and `sloc` are removed from the `kwargs` if present to avoid running into collisions?

Reply:
Ah, interesting point. 

I think if we simply replicate a similar setup to what the old code did, we'd avoid the issue and enable users to properly override everything.

So:
```python
dftt_args = dict(tr=0, sloc=[...]) | kwargs
dftt = DFTTest.denoise(clip, **dftt_args)
```

(Also, it looks like we're mixing `dftt` and `dfft` a bit in this code (I do it all the time)).

I'll make this update.